products/envoy.md

```diff
@@ -42,6 +42,12 @@ auto:
 # eol(X>=1.12) can be found on https://github.com/envoyproxy/envoy/blob/main/RELEASES.md#major-release-schedule
 # eol(X<1.12) = releaseDate(X+1)
 releases:
+-   releaseCycle: "1.33"
+    releaseDate: 2025-01-14
+    eol: false # tbd on https://github.com/envoyproxy/envoy/blob/main/RELEASES.md#major-release-schedule
+    latest: '1.33.0'
+    latestReleaseDate: 2025-01-14
+
 -   releaseCycle: "1.32"
     releaseDate: 2024-10-15
```

Comment:
could you make this matches with 1.33's release date . whether keep this one same but change others or just change this one

Reply:
I am not sure if I understand, why do I need to change it?

Reply:
opps i misread it , i was thinking 1.32 became eol in same day of releasing 1.33